src/components/App.tsx

```diff
@@ -67,7 +68,7 @@ function App() {
             />
             <Route
               path='/beta/compare-results'
-              element={<CompareResultsView theme={protocolTheme} />}
+              element={<ResultsView />}
```

Comment:
How will dark mode be handled in the Results view?

Reply:
Or would you like me to make a ticket to do so when the results view is done?

Reply:
Fixed, added the dark mode now.